frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -2151,3 +2158,23 @@ func reportUnusedFromArgs(unmatched map[string]struct{}, location []parser.Range
 		linter.RuleUndeclaredArgInFrom.Run(warn, location, msg)
 	}
 }
+
+type parserLocation struct {
+	Loc   []parser.Range
+	IsSet bool
+}
+
+func (v *parserLocation) Set(loc []parser.Range) {
```

Comment:
Maybe something like `* instructionTracker) MarkUsed(loc []parser.Range)` ?

Reply:
Seems like a better name to me. Done.